vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/rules/EliminateUnnestAggregateSubplanRule.java

```diff
@@ -78,6 +78,7 @@ public boolean rewritePre(Mutable<ILogicalOperator> opRef, IOptimizationContext
             return false;
         }
         AbstractFunctionCallExpression functionCall = (AbstractFunctionCallExpression) logicalExpression;
+
```

Comment:
Please remove this white space. Try to keep your change only to the files you edit.